docs/installation.md

```diff
@@ -1,27 +1,132 @@
 # Installation
 
-**Ergonode** is an extensive application with several repositories. <br>
-The system itself is divided into 2 parts - [**frontend**][front] and [**backend**][back], which work independently of each other. <br>
-This may complicate the installation process, but we have created a precise installation guide every time our application is used.<br>
-
-
-1. If you are a developer who is interested in **open-source** projects, you're in the right place because **Ergonode** is such a project.<br>
-You will be very interested in installing the application this way:<br>
- - [Frontend contribution installation][cf]
- - [Backend contribution installation][cb]
-
-2. You are our implementation partner and you are looking for instances of how to quickly build an application skeleton for your client.<br>
-You will be very interested in installing the application in such a way:
- - [Frontend partner installation][pf]
- - [Backend partner installation][pb]
-
-3. We also have for you a docker repository with our project, which will certainly make the work easier for many people:
-  - [Docker installation][docker]
-
-[front]: https://github.com/ergonode/frontend
-[back]: https://github.com/ergonode/backend
-[cf]: installation/frontend-contribution
-[pf]: installation/frontend-development
-[cb]: installation/backend-contribution
-[pb]: installation/backend-development
-[docker]: installation/docker
\ No newline at end of file
+**Ergonode** is a headless system which uses two different application [**frontend**][frontend] and [**backend**][backend]. 
```

Comment:
Maybe we should add something like this at the beginning: 
"This is the initial installation that allows you to run the application quickly to familiarize yourself with it. The installation runs the application in production mode so to contribute to the project see <link>. "